SR2024/2023-09-23-place-confirmation.md

```diff
@@ -0,0 +1,51 @@
+---
+to: Student Robotics 2024
+subject: Confirming your place at Student Robotics 2024
+---
+
+You're in! We're happy to confirm you've got a place at Student Robotics 2024!
+
+As the new school year starts, so does Student Robotics. The year starts with a
+"Kickstart" event which gives competitors and team supervisors an introduction
+to this year's competition and overview of how to build and engineer their
+robot.
+
+At various points in the year we will have a number of "Tech Days" at locations
+around the UK. These provide an opportunity for your team to get hands-on help
+from our volunteer mentors, make progress on their robot and meet other teams.
+
+Throughout the year your team will be able to get support for our mentors via
+[Discord](https://studentrobotics.org/docs/tutorials/discord). In addition to
+shared channels where any team can ask questions, each team will have their own
+private text chat channel and team supervisors can request voice & video
+channels too. These offer a space for teams to discuss their robot and get
+direct support from our mentors at any time. We will send out invites to Discord
+separately.
+
+This year **Kickstart is fully virtual**. We strongly encourage teams to meet up
+to watch the presentations, get to know each other and start working together on
+the kit. We will be shipping kits out to teams ahead of the event, so
+_please ensure we have the correct shipping address_.
+
+On the day we will be livestreaming an introduction to the competition, the game
+and the kits. We will also be providing a set of tasks which will familiarise
+competitors with the kit, which the teams can complete afterwards. There will be
+mentors around in Discord throughout the day to support teams working through
```

Comment:
Issue: We've currently not defined how long mentors will be around. We should add some time frames here and to the website.

Reply:
I don't think that adding those details here is useful. The goal of this email is mostly to complete the signup process. The additional information about the year & kickstart is mostly to help any rookies place their signup in context; the secondary goal of this information is to advertise kickstart to teams, giving them an idea of what they need to do for it and what they need to prepare.

Those details definitely should go on the website and we can (of course) send out more details closer to the event. I imagine that the next email might for example include both the kickstart details and the Discord sign-in details.

Reply:
True, but there's no reason _not_ to send it in this email. "Throughout the day" can mean different things to different people.

Reply:
> there's no reason not to send it in this email.

Yes, there's a huge reason not to -- we don't have this information yet! Holding back emails in the hope that we can include just one more detail is something that we're _terrible_ at doing and leads to a lack of communications. Sending more emails with more details later (even if we duplicate information already sent) would be fine, beneficial even.

I appreciate that there are a number of interpretations here, though I hope that everyone will interpret this as meaning that this is a mostly-all-day event rather than a one-hour presentation. This is the main thing I'm trying to convey here.

(A smaller reason might be that it's distracting information that people don't need yet and are likely to forget anyway).